components/eam/src/chemistry/yaml/aero_model/f90_yaml/aero_model_wetdep_end_yml.f90

```diff
@@ -0,0 +1,70 @@
+#ifdef YAML_AERO_MODEL_WETDEP
+  if(yaml%flag_print) then ! if this column exists in lchnk
+
+     !write output header
+     call write_output_header(unit_output)
+
+     !start writing data
+
+        ! write out fields that are set within intent-out structure 'cam_out'
+
+        call write_var(unit_output,'cam_out_bcphiwet',cam_out%bcphiwet(yaml%col_print))
+        call write_var(unit_output,'cam_out_bcphidry',cam_out%bcphidry(yaml%col_print))
+        call write_var(unit_output,'cam_out_ocphiwet',cam_out%ocphiwet(yaml%col_print))
+        call write_var(unit_output,'cam_out_ocphidry',cam_out%ocphidry(yaml%col_print))
+        call write_var(unit_output,'cam_out_dstwet1',cam_out%dstwet1(yaml%col_print))
+        call write_var(unit_output,'cam_out_dstwet2',cam_out%dstwet2(yaml%col_print))
+        call write_var(unit_output,'cam_out_dstwet3',cam_out%dstwet3(yaml%col_print))
+        call write_var(unit_output,'cam_out_dstwet4',cam_out%dstwet4(yaml%col_print))
+
+        ! ptend is passed as intent-out to aero_model_wetdp, so only include as output data.
+        ! The following extract the fields from ptend used as output and write them to the data files.
+        ! Use -9999.9 to fill in unused portions of the constituent arrays
+
+        if (allocated(ptend_lq_1darr)) deallocate(ptend_lq_1darr)
+        allocate(ptend_lq_1darr(size(ptend%lq)))
+
+        if (allocated(ptend_q_3darr)) deallocate(ptend_q_3darr)
+        allocate(ptend_q_3darr(size(ptend%q,1),size(ptend%q,2),size(ptend%q,3)))
+
+        do imm=1,size(ptend%lq)
+           if (imm<=15) then
+                ptend_lq_1darr(imm) = -9999.9
+                ptend_q_3darr(:,:,imm) = -9999.9
+           else
+                ptend_lq_1darr(imm) = ptend%lq(imm)
+                ptend_q_3darr(:,:,imm) = ptend%q(:,:,imm)
+           endif
+        enddo
+        call write_var(unit_output,'ptend_lq',ptend_lq_1darr(:))
+        call write_var(unit_output,'ptend_lq',ptend_q_3darr(yaml%col_print,yaml%lev_print,:))
+        deallocate(ptend_lq_1darr)
+        deallocate(ptend_q_3darr)
+
```

Comment:
We also need to write the following variables as outputs:
- `dgncur_a`
- `wetdens`
- `qaerwat`
- `dgnumwet`
- `fracis`

As a  reference see [this doc](https://eagles-project.atlassian.net/wiki/spaces/Computation/pages/1732968449/Integrating+wet+scavenging+process+in+EAMxx).

Reply:
These have been added.